.github/workflows/validate.yml

```diff
@@ -16,7 +16,21 @@ jobs:
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.13, 10, 12, 14]
+        node: [10.13, 10, 12.13, 12, 14, 15]
+        gatsby: [2.20, 2, 3.0, 3]
+        exclude:
+          - gatby: 2.20
+            node: 15
+          - gatby: 2
+            node: 15
+          - gatby: 3.0
+            node: 10.13
+          - gatby: 3.0
+            node: 10
+          - gatby: 3
+            node: 10.13
+          - gatby: 3
+            node: 10
```

Comment:
Technically I can support both `gatsby@^2.20.0` & `gatsby@^3.0.0`, but this will create 18 CI runs every time.

If I drop gatsby 2.x support (like I did in https://github.com/MichaelDeBoey/gatsby-plugin-instagram-embed/pull/16), there will only be 4 CI runs but that will be a breaking change.

@kentcdodds What do you think? 🤔

Reply:
I'm addition to dropping node 10, you could also just verify gatsby v2 works in a single version of Node (14 for example).